dao/src/integ-test/resources/uk/gov/moj/sdt/dao/sql/PurgeDaoTest.sql

```diff
@@ -1,3 +1,11 @@
+TRUNCATE TABLE global_parameters CASCADE;
+TRUNCATE TABLE target_applications CASCADE;
+TRUNCATE TABLE bulk_customers CASCADE;
+TRUNCATE TABLE service_requests CASCADE;
+TRUNCATE TABLE bulk_submissions CASCADE;
+TRUNCATE TABLE individual_requests CASCADE;
+TRUNCATE TABLE error_logs CASCADE;
```

Comment:
No data is inserted into the TARGET_APPLICATIONS, BULK_CUSTOMERS or SERVICE_REQUESTS tables, so do they need to be truncated?

Reply:
data is inserted into these tables.. Please see line numbers 21 for example